Change the Debian section and add unbuntu (#740)
hyprwm/hyprland-wiki: pages/Getting Started/Installation.md

{{% /details %}}
{{% details title="Debian*" closed="true" %}}

Hyprland recently made it into the SID and trixie repos and can be installed with

```sh
sudo apt install hyprland
```

> Note: Even though that Hyprland is in the trixie repos, it is still recommended to install from SID as some dependencies are out dated.

Reviewer: ```suggestion
Note: Even though Hyprland is in the trixie repos, it is still recommended to install from SID, as some dependencies in the trixie repo are outdated.
```

{{% /details %}}

{{% details title="Ubuntu*" closed="true" %}}
Hyprland made it into the universe repos and can be installed with 
```bash
sudo add-apt-repository universe && sudo apt-get update && sudo apt-get install -y hyprland
```

> Build dependencies:

Reviewer: Can you add a newline after the `{{% details` line and before and after the codeblock?

Please note that since Ubuntu is generally behind with dependencies, it's not
guaranteed that the build process will work at all. Even if it is, it's likely

> that it will break at some point in the future.  **Always use the latest version of Ubuntu for the most up to date dependencies if building or installing.** Note: Your mileage may also differ as GDM does have sone bugs with Hypr check the master tutorial for more info.

Reviewer: ```suggestion
that it will break at some point in the future.

{{< callout >}}

Always use the latest version of Ubuntu for the most up to date dependencies.

Note: Your mileage may vary, as GDM has some bugs with Hyprland. Check the [Master Tutorial](../Master-Tutorial) for more info.

{{< /callout >}}
```
